proto/storage/collector_runtime_config.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package storage;
+
+import "storage/resource_collection.proto";
+
+option go_package = "storage";
+option java_package = "io.stackrox.proto.storage";
+
+message CollectorRuntimeConfigACK {
+  string instance_id = 1;
+  Action action = 2;
```

Comment:
Maybe we can add a troubleshooting field with the reason for a failure ? Sensor might be able to stuff this in a diagnostic bundle.